portfolio/src/main/webapp/script.js

```diff
@@ -287,18 +288,51 @@ var styledMapType = new google.maps.StyledMapType(
     ],
             {name: 'Styled Map'});
 
+    // The location of Nashville
+    var nashville = {lat: 36.164347196601156, lng: -86.78117037227406};
+
     // Create a map object, and include the MapTypeId to add
     // to the map type control.
+    // The map, centered at Nashville
     var map = new google.maps.Map(document.getElementById('map'), {
-        center: {lat: 36.164347196601156, lng: -86.78117037227406},
-        zoom: 14,
+        center: {lat: 38, lng: -88},
```

Comment:
I'm still wondering why this isn't `Math.round(nashville.lat)` and `Math.round(nashville.lng)`, but this still looks good nonetheless. I'm assuming you've run it and it centers the map the way you like it.

Reply:
yup, the center is adjusted to also show location markers in CO, CT, and MA